### ISSUE TITLE
licence

### ISSUE THREAD

**Issue by ennswi** [gggard/AndroidCaldavSyncAdapater#255]
Hello, 
I'm registered that they're is no licence in this repo.
On this reason is that what we do, stealing your code.

Is it right, that I set my fork under the GPL v3?
Or what can i do?

**Comment by A3K** [comment]
It is GPL3 https://github.com/gggard/AndroidCaldavSyncAdapater/commit/9c18af66d9b0a998e37927ba5b59c5ad15d787d3

**Comment by ennswi** [comment]
Youre right. But in this case, are only the files with this header are under GPL v3.

If it is OK for @wildgarden and @gggard 
I'm creating an licence.MD file with the GPL v3 implemented.

**Comment by gggard** [comment]
Dear all,

Yes GPL 3 was m'y intention, contributions are welcomed !

G

Le 19 juillet 2016 09:14:33 GMT+02:00, ennswi notifications@github.com a écrit :

> Youre right. But in this case, are only the files with this header are
> under GPL v3.
> 
> If it is OK for @wildgarden and @gggard 
> I'm creating an licence.MD file with the GPL v3 implemented.
> 
> ---
> 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/gggard/AndroidCaldavSyncAdapater/issues/255#issuecomment-233550530

## 

Envoyé de mon téléphone. Excusez la brièveté.

**Comment by wildgarden** [comment]
I'm absolutly fine with it.

**Comment by rekire** [comment]
Just my thoughts: I love open source and I also published some libs, but why in hell GPL? You cannot use it in closed apps, since you will be forced to publish the full source code even if that lib is a very small part.

**Comment by gggard** [comment]
Dear rekine, 

GPL exactly for the reason you invoke, that kind of application is not a lib... 
you like it ? Contribute !
 you believe it can be part of a larger piece ? Perfect, open source that tremendous app!

Gege

Le 1 septembre 2016 06:58:28 GMT+02:00, rekire notifications@github.com a écrit :

> Just my thoughts: I love open source and I also published some libs,
> but why in hell GPL? You cannot use it in closed apps, since you will
> be forced to publish the full source code even if that lib is a very
> small part.

## 

Envoyé de mon téléphone. Excusez la brièveté.
